src/components/menu/GitBranchMenu.vue

```diff
@@ -234,47 +234,16 @@ async function initBranches() {
 }
 
 /**
- * Send event to open the GitNewBranchDialog.
+ * Send event to open the dialog corresponding to the key and close the menu..
```

Comment:
remove one of the dote at the ends of the line

Reply:
done